src/simularium/VisGeometry.ts

```diff
@@ -951,6 +955,12 @@ class VisGeometry {
     }
 
     public setColorForIds(ids: number[], colorId: number): void {
+        if (this.isIdColorMappingSet) {
+            throw new FrontEndError(
+                "Attempted to set agent-color after color mapping was finalized"
+            );
```

Comment:
Is this being thrown as often as we were seeing all yellow agents before? This doesn't seem like a helpful/actionable message to the front end user. If I understand this correctly, I think this should just silently return out.

Reply:
No, this is thrown if a dev calls finalize, then attempts to set the color. There shouldn't be a way for a user to cause this through the simularium-ui, this error was to let us know quickly if we use the interface in a way that isn't intended.

On second look, it would make more sense to print this out to the console than throw a FE error. I'll update the PR.

Reply:
This function is currently only called when the trajectory file info arrives (and the render loop references the results)